Add scenario workflow for using Cert Manager to manage certificates (#86)
confluentinc/confluent-kubernetes-examples: security/using-cert-manager/README.md

> # Use Cert Manager to provide certificates for use

Reviewer: ```suggestion
# Use Cert Manager to provide certificates
```

The provided `certificate.yaml` is configured  with namespace `confluent` and an opinionated name choices for Confluent Platform.

Check issuers: `kubectl -n confluent get issuers`

Check certificates: `kubectl -n confluent get certificates`

### Cert-Manager with Let's Encrypt Issuer

Follow the instructions in https://cert-manager.io/docs/configuration/acme/ to set this up with your account for Let's Encrypt.

Once that is set up, you'll need to modify and use the Kustomize bundle in `$TUTORIAL_HOME/cert-manager/ca`.

## Deploy CP platform

Before deploying the Confluent Platform components, make sure all the secrets objects are created by running
`kubectl -n confluent get secrets`. 

The secret names can be checked with output of `kubectl -n confluent get certificates`.

The Confluent Platform spec in `$TUTORIAL_HOME/resources/confluent-platform.yaml` is using these generated secrets 
through cert-manager to configure TLS encryption for all network communication. In this example spec, no authentication 
is configured.

```
kubectl apply -n confluent -f $TUTORIAL_HOME/resources/confluent-platform.yaml
```

## Validate

### Status

> Use the status section to get information about the component: like endpoints, internal topics etc. Here, you'll see 

Reviewer: ```suggestion
Use the status section to get information about the component, such as endpoints, internal topics, etc. Here, you'll see 
```

that each component endpoint is TLS enabled.

```
kubectl -n confluent get kafka -oyaml
```

```
kubectl -n confluent get zookeeper -oyaml
kubectl -n confluent get schemaregistry -oyaml
kubectl -n confluent get ksqldb -oyaml
kubectl -n confluent get connect -oyaml
kubectl -n confluent get controlcenter -oyaml
```

> ### ControlCenter UI

Reviewer: ```suggestion
### Control Center UI
```
